telegrambots-meta/src/main/java/org/telegram/telegrambots/meta/api/methods/payments/star/EditUserStarSubscription.java

```diff
@@ -40,7 +41,7 @@ public class EditUserStarSubscription extends BotApiMethodBoolean {
      */
     @JsonProperty(USER_ID_FIELD)
     @NonNull
-    private Integer userId;
+    private Long userId;
```

Comment:
This is not needed to be a Long, as per docs:

_It is safe to use 32-bit signed integers for storing all Integer fields unless otherwise noted._